static/js/dropdown_list_widget.js

```diff
@@ -154,6 +154,10 @@ DropdownListWidget.prototype.dropdown_focus_events = function () {
                 }
                 break;
             }
+            case "Escape": {
+                dropdown_menu.dropdown("toggle");
+                break;
+            }
```

Comment:
This does not work correctly with bot-owner widget in the `Change bot info and owner` modal opened from settings, it closes the bot info modal. Works fine in other places.